lib/client-assets.php

```diff
@@ -78,6 +78,13 @@ function gutenberg_register_scripts_and_styles() {
 	// WordPress packages.
 	wp_register_script( 'wp-tinymce', includes_url( 'js/tinymce/' ) . 'wp-tinymce.php', array() );
 
+	wp_register_script(
+		'wp-autop',
```

Comment:
For some reason, autop was not a separate script but was bundled.

Reply:
`url` and `wordcount` are also bundled into something, I bet `editor` or `edit-post`.